.gitignore

```diff
@@ -20,3 +20,5 @@ events/
 
 # MacOS makes this one
 .DS_Store
+
+.repository
```

Comment:
Curious: I hoped my google skillz were better, but I didn't manage to find what .repository files are. Would you mind sharing that @jangalinski?

Reply:
a personal "trick" ... I add a symlink to the project that points to the .m2 repository path of the axon framework. Makes it easier to check/delete when testing mvn install.